test/unit/server-statefulset.bats

```diff
@@ -160,6 +160,128 @@ load _helpers
   [ "${actual}" = "foo" ]
 }
 
+#--------------------------------------------------------------------
+# exposeGossipAndRPCPorts
+
+@test "server/StatefulSet: server gossip and RPC ports are not exposed by default" {
+  cd `chart_dir`
+  local object=$(helm template \
+      -s templates/server-statefulset.yaml  \
+      . | tee /dev/stderr )
+
+  # Test that hostPort is not set for gossip ports
+  local actual=$(echo "$object" |
+      yq -r '.spec.template.spec.containers[0].ports[1].hostPort' | tee /dev/stderr)
```

Comment:
can we use a `select` on `ports` to pull out the port by `name`? Then we don't rely on the index like `ports[1]`.

Reply:
Ahh good idea!